tfs/src/main/java/hudson/plugins/tfs/model/AbstractHookEvent.java

```diff
@@ -112,7 +116,27 @@ static JSONObject fromResponseContributors(final List<GitStatus.ResponseContribu
                     return new TeamEventsEndpoint.ScheduledResponseContributor(project);
                 }
 
-                final TeamPushTrigger pushTrigger = TeamEventsEndpoint.findTrigger(job, TeamPushTrigger.class);
+                boolean shouldRun = false;
+                FlowDefinition jobDef = ((WorkflowJob)job).getDefinition();

```

Comment:
jobDef should be final
before you cast job to WorkflowJob there should be a check to make sure the cast will work.